Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -439,6 +439,11 @@ public void HandlePacket(Client client, Player player, ChatMsgPacket packet)
                 return;
             }
 
+            if(Options.LogChatMessages)
+            {
+                Log.Pretty.Info("Client " + client.GetIp() + " Player: " + player.Name + " Message: " + packet.Message);
```

Comment:
This should use `$"{variable}"` syntax and also simplify the message to save space:

`$"{client.GetIp()} {player.Name}: {packet.Message}"`

Reply:
Sounds good